### ISSUE TITLE
fcos' performance is far from the paper

### ISSUE THREAD

**Issue by DuZzzs** [TuSimple/simpledet#289]
The results of my training are far from the official [code.](https://github.com/tianzhi0549/FCOS)

modified：
```
backbone：resnet_v1b_50
train dataset: coco2017
epoch: 7
device：TITAN RTX
```
**results**: test on coco minival2014.
my result :&emsp;&emsp;map = 34.7, &emsp;inference time: 167 ms/pic
official result :  map = 37.1, &emsp;inference time:  50 ms/pic 
Is this a problem of implementation or my operation?

**Comment by huangzehao** [comment]
@kfxw

**Comment by kfxw** [comment]
@DuZzzs 
Hi!
The speed of inference code is not optimized. Some python custom ops are used and they are the bottlenecks. Only the training speed is guaranteed in this code.
As for the accuracy, I used 'coco_train2014 + coco_valminusminival2014' for training. This is the common practice. I remember the mAP fluctuated between around 36.8 to 37.2. You can try this training setting.

**Comment by DuZzzs** [comment]
@kfxw 
ok. Thank you very much.
